src/main/java/com/contentful/java/cma/ModuleSpaceMemberships.java

```diff
@@ -0,0 +1,287 @@
+   *
+   * @param spaceId      the space this membership is hosted by.
+   * @param membershipId the id of the membership to be found.
+   * @return null if no membership was found, otherwise the found membership.
+   * @throws IllegalArgumentException if space id is null.
+   * @throws IllegalArgumentException if membership id is null.
+   */
+  public CMASpaceMembership fetchOne(String spaceId, String membershipId) {
+    assertNotNull(spaceId, "spaceId");
+    assertNotNull(membershipId, "membershipId");
+
+    return service.fetchOne(spaceId, membershipId).toBlocking().first();
+  }
+
+  /**
+   * Create a new membership.
+   *
+   * @param spaceId    the space id to host the membership.
+   * @param membership the new membership to be created.
+   * @return the newly created membership.
+   * @throws IllegalArgumentException if space id is null.
+   * @throws IllegalArgumentException if membership is null.
+   */
+  public CMASpaceMembership create(String spaceId, CMASpaceMembership membership) {
+    assertNotNull(spaceId, "spaceId");
+    assertNotNull(membership, "membership");
+
+    final CMASystem sys = membership.getSystem();
+    membership.setSystem(null);
+
+    try {
+      return service.create(spaceId, membership).toBlocking().first();
+    } finally {
+      membership.setSystem(sys);
+    }
+  }
+
+  /**
+   * Update the given membership instance on Contentful.
+   * <p>
+   * Please make sure that the instance provided is fetched from Contentful. Otherwise you will
+   * get an exception thrown.
+   *
+   * @param spaceId    the id of the space to be used.
+   * @param membership the membership fetched from contentful, updated by caller, to be updated.
+   * @return the updated membership.
+   * @throws IllegalArgumentException if space id is null.
+   * @throws IllegalArgumentException if membership is null.
+   * @throws IllegalArgumentException if membership id is null.
+   * @throws IllegalArgumentException if membership does not have a version attached.
+   */
+  public CMASpaceMembership update(String spaceId, CMASpaceMembership membership) {
+    assertNotNull(spaceId, "spaceId");
+    assertNotNull(membership, "membership");
+
+    final String id = getResourceIdOrThrow(membership, "membership");
+    final Integer version = getVersionOrThrow(membership, "update");
+
+    final CMASystem sys = membership.getSystem();
+    membership.setSystem(null);
+
+    try {
+      return service.update(spaceId, id, membership, version).toBlocking().first();
+    } finally {
+      membership.setSystem(sys);
+    }
+  }
+
+  /**
+   * Delete the given membership instance.
+   * <p>
+   * Please make sure that the instance provided is fetched from Contentful. Otherwise you will
+   * get an exception thrown.
+   *
+   * @param spaceId    the id of the space to be used.
+   * @param membership the membership fetched from contentful, updated by caller, to be deleted.
+   * @return an empty string on success.
```

Comment:
Yes, that is a great spot! I forgot to update after some iteration the documentation! (Documentation driven developement for the win!)